packages/test/src/load/starter.ts

```diff
@@ -126,22 +127,33 @@ async function main() {
   const taskQueue = getRequired(args, '--task-queue');
 
   const connection = new Connection({ address: serverAddress });
-
   const client = new WorkflowClient(connection.service, { namespace });
-
   const stopCondition = runForSeconds ? new UntilSecondsElapsed(runForSeconds) : new NumberOfWorkflows(iterations);
 
-  const passed = await runWorkflows({
-    client,
-    workflowName,
-    taskQueue,
-    stopCondition,
-    concurrency: concurrentWFClients,
-    minWFPS,
-  });
-  if (!passed) {
-    console.error('Load test did not pass');
-    process.exit(1);
+  let workflowsToRun: string[];
+  if (workflowName === 'yummy-sampler-mode') {
+    // Special workflow alias to run many different load tests sequentially.
+    // Expected wf/sec should be set low since some of these by their nature have
+    // higher latency.
+    workflowsToRun = ['cancelFakeProgress', 'childWorkflowCancel', 'childWorkflowSignals', 'cancellationScopes'];
```

Comment:
`cancellationScopes` isn't that interesting as a load test, it's more a test of the framework code.
`childWorkflowCancel` and `childWorkflowSignals` are better used as integration tests as they test multiple scenarios but I'd leave them for now since they provide pretty good coverage for child workflows, we can replace them later.
We should add a workflow with a query, that's missing here.

Reply:
OK I will remove scopes and add a smoke test that does a little bit of everything.

This is really more like integ tests with a bunch of them in parallel. It's really more of a smoke test than a proper load test but I think it's reasonable to just have it here as this general test runner.